packages/console/src/Console.tsx

```diff
@@ -251,6 +253,10 @@ export class Console extends PureComponent<ConsoleProps, ConsoleState> {
     if (props.objectMap !== prevProps.objectMap) {
       this.updateObjectMap();
     }
+
+    if (state.isStuckToBottom) {
+      this.scrollConsoleHistoryToBottom();
+    }
```

Comment:
You should be able to remove the calls to `scrollConsoleHistoryToBottom` in the 2 spots it's called w/o force and then actually drop the `force` param I think since everywhere would be calling it w/ force.

The function as written is really `queueScrollConsoleHistoryToBottomIfCurrentlyAtBottomOrForced`

So instead the function could just be something like this

```ts
scrollConsoleHistoryToBottom(): void {
  const pane = this.consoleHistoryScrollPane.current;
  if (pane == null) {
    return;
  }
  pane.scrollTo({ top: pane.scrollHeight });
```